### PR TITLE
Feature/formik examples

### PR DESCRIPTION
**Pull request by brauliodiez**

### CODE REVIEW

File: examples/formik/js/async-validator/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  <div className="app">
+    <h1>
+      Basic{' '}
+      <a
+        href="https://github.com/jaredpalmer/formik"
+        target="_blank"
+        rel="noopener"
+      >
+        Formik
+      </a>{' '}
```
Comment (Nasdan) [inline examples/formik/js/async-validator/index.js]: Remove `{' '}`

File: examples/formik/js/async-validator/index.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
+      Basic{' '}
```
Comment (Nasdan) [inline examples/formik/js/async-validator/index.js]: here too

File: examples/formik/js/async-validator/helper.css [hunk 3]
```diff
@@ -0,0 +1,91 @@
+* {
```
Comment (Nasdan) [inline examples/formik/js/async-validator/helper.css]: Remove it. * has not specifity

File: examples/formik/js/async-validator/helper.css [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  box-sizing: border-box;
+  font-family: ;
```
Comment (Nasdan) [inline examples/formik/js/async-validator/helper.css]: This is empty

File: examples/formik/js/async-validator/package.json [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    "react"
+  ],
+  "main": "index.js",
+  "dependencies": {
+    "@lemoncode/fonk": "latest",
+    "@lemoncode/fonk-formik": "latest",
+    "formik": "latest",
+    "react": "latest",
+    "react-dom": "latest",
+    "react-final-form": "latest",
```
Comment (Nasdan) [inline examples/formik/js/async-validator/package.json]: remove not needed dependency

File: examples/formik/js/async-validator/package.json [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+    "styled-components": "latest"
```
Comment (Nasdan) [inline examples/formik/js/async-validator/package.json]: remove too

File: examples/formik/js/async-validator/styles.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
+import styled, { css } from 'styled-components';
```
Comment (Nasdan) [inline examples/formik/js/async-validator/styles.js]: Not used file

File: examples/formik/js/async-validator/validate.js [hunk 8]
```diff
@@ -0,0 +1,11 @@
+import { Validators } from '@lemoncode/fonk';
```
Comment (Nasdan) [inline examples/formik/js/async-validator/validate.js]: The name of this file in `react-final-form` examples is `form-validation.js`. Rename (one of them) to get same file name?

File: examples/formik/js/basic/helper.css [hunk 9]
```diff
@@ -0,0 +1,91 @@
+* {
```
Comment (Nasdan) [inline examples/formik/js/basic/helper.css]: same here

File: examples/formik/js/basic/index.js [hunk 10]
```diff
@@ -0,0 +1,85 @@
+// Helper styles for demo
+import './helper.css';
+import { MoreResources, DisplayFormikState } from './helper.css';
```
Comment (Nasdan) [inline examples/formik/js/basic/index.js]: remove this import?

File: examples/formik/js/basic/index.js [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
+      Basic{' '}
```
Comment (Nasdan) [inline examples/formik/js/basic/index.js]: same here

File: examples/formik/js/basic/index.js [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+      <a
+        href="https://github.com/jaredpalmer/formik"
+        target="_blank"
+        rel="noopener"
+      >
+        Formik
+      </a>{' '}
```
Comment (Nasdan) [inline examples/formik/js/basic/index.js]: same here

File: examples/formik/js/async-validator/index.js [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+            dirty,
+            isSubmitting,
+            handleChange,
+            handleBlur,
+            handleSubmit,
+            handleReset,
+          } = props;
+          return (
+            <form onSubmit={handleSubmit}>
+              <label htmlFor="user" style={{ display: 'block' }}>
```
Comment (Nasdan) [inline examples/formik/js/async-validator/index.js]: Not neede style isn't it? We have it in `helper.css` file

File: examples/formik/js/basic/index.js [hunk 14]
```diff
@@ -0,0 +1,85 @@
...
+            dirty,
+            isSubmitting,
+            handleChange,
+            handleBlur,
+            handleSubmit,
+            handleReset,
+          } = props;
+          return (
+            <form onSubmit={handleSubmit}>
+              <label htmlFor="email" style={{ display: 'block' }}>
```
Comment (Nasdan) [inline examples/formik/js/basic/index.js]: same here

File: examples/formik/js/async-validator/index.js [hunk 15]
```diff
@@ -0,0 +1,84 @@
...
+import './helper.css';
+
+import React from 'react';
+import { render } from 'react-dom';
+import { Formik } from 'formik';
+import { formValidation } from './validate';
+
+const App = () => (
+  <div className="app">
+    <h1>
```
Comment (Nasdan) [inline examples/formik/js/async-validator/index.js]: Why is it all html inside an `h1` element?

File: examples/formik/js/basic/package.json [hunk 16]
```diff
@@ -0,0 +1,25 @@
...
+    "react"
+  ],
+  "main": "index.js",
+  "dependencies": {
+    "@lemoncode/fonk": "latest",
+    "@lemoncode/fonk-formik": "latest",
+    "formik": "latest",
+    "react": "latest",
+    "react-dom": "latest",
+    "react-final-form": "latest",
```
Comment (Nasdan) [inline examples/formik/js/basic/package.json]: same here

File: examples/formik/js/basic/styles.js [hunk 17]
```diff
@@ -0,0 +1,129 @@
+import styled, { css } from 'styled-components';
```
Comment (Nasdan) [inline examples/formik/js/basic/styles.js]: same here

File: examples/formik/js/formik-components/readme.md [hunk 18]
```diff
@@ -0,0 +1,17 @@
+# Formik Components
+
+Example integrating `fonk` with `formik`.
+
+Using Field Formik component.
+
+[![See validate-field example](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/github/lemoncode/fonk/tree/master/examples/react-final-form/js/formik-component)
```
Comment (Nasdan) [inline examples/formik/js/formik-components/readme.md]: Typo `formik-components`. The folder name is with `s`

File: examples/formik/js/async-validator/helper.css (hunk 3 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/helper.css]: Is just the example from formik official site, I would keep it like it is, Maybe we should open a PR to them :),

File: examples/formik/js/async-validator/helper.css (hunk 4 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/helper.css]: You are right, but going to skip updates on this CSS, is from formik examples

File: examples/formik/js/async-validator/index.js (hunk 15 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/index.js]: Is from Formik Examples, going to add some meaningful text

File: examples/formik/js/async-validator/index.js (hunk 2 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/index.js]: updating H1

File: examples/formik/js/async-validator/index.js (hunk 1 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/index.js]: Is just from formik examples, I want to keep them as similar as they are in their website... good news is that they have room from improvement :)

File: examples/formik/js/async-validator/index.js (hunk 13 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/index.js]: Same thing formik examples stuff

File: examples/formik/js/async-validator/package.json (hunk 5 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/package.json]: Good catch

File: examples/formik/js/async-validator/package.json (hunk 6 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/package.json]: Good catch

File: examples/formik/js/async-validator/styles.js (hunk 7 above)
Comment (brauliodiez) [reply examples/formik/js/async-validator/styles.js]: removed, thanks

File: examples/formik/js/basic/index.js (hunk 10 above)
Comment (brauliodiez) [reply examples/formik/js/basic/index.js]: Right, thanks

File: examples/formik/js/basic/package.json (hunk 16 above)
Comment (brauliodiez) [reply examples/formik/js/basic/package.json]: done

File: examples/formik/js/formik-components/readme.md (hunk 18 above)
Comment (brauliodiez) [reply examples/formik/js/formik-components/readme.md]: Good point and replace react-final-form with formik. ouch :), thanks

**Pull request merged**